presto-tests/src/test/resources/queue_config_dashboard.json

```diff
@@ -17,8 +17,8 @@
     {
       "source": "(?i).*dashboard.*",
       "queues": [
-        "user.${USER}",
         "dashboard.${USER}",
+        "user.${USER}",
```

Comment:
Maybe this is rather irrelevant to this particular pull request. But having a rule like this will have unintuitive behavior? Specifically, if the user submits 3 dashboard requests, now he/she can't run any interactive queries.